code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -558,7 +559,7 @@ var/global/list/damage_icon_parts = list() //see UpdateDamageIcon()
 
 	//Shoes can be affected by uniform being drawn onto them
 	update_inv_shoes()
-	
+

```

Comment:
A Polaris .dm file

Reply:
I don't really even know what's doing those. Some sorta setting in the editor I've somehow managed to hit?